pom.xml

```diff
@@ -73,6 +73,10 @@
             <artifactId>liquibase-maven-plugin</artifactId>
             <version>4.22.0</version>
         </dependency>
+        <dependency>
+            <groupId>org.hibernate.validator</groupId>
```

Comment:
In Spring Boot it's better to choose starters over concrete dependencies, so I suggest to use validation starter